Project.toml

```
@@ -1,9 +1,9 @@
 name = "AbstractMCMC"
 uuid = "80f14c24-f653-4e6a-9b94-39d6b0f70001"
-keywords = ["markov chain monte carlo", "probablistic programming"]
+keywords = ["markov chain monte carlo", "probabilistic programming"]
 license = "MIT"
 desc = "A lightweight interface for common MCMC methods."
-version = "5.2.0"
+version = "5.2.1"
```

I think it's not a bug fix but rather a feature, so I'd make it
```suggestion
version = "5.3.0"
```

---

Ooh, okay, I thought of it more as a bug fix ('bug' being that error message is unclear). But thinking about it a bit more this does modify the package's API in that people can catch the error and check the message, so will bump the minor.